### DISCUSSION TITLE
Procedure on adding alpha support for Maltese

### DISCUSSION THREAD

**Discussion by drvenabili** [explosion/spaCy#11183] [Language Support]
Hi!

Following [this](https://github.com/explosion/spaCy/discussions/3056#discussioncomment-3118715), I am opening a new discussion. I would like to contribute alpha support for Maltese (ISO 639-1 `mt`). 
Do I understand correctly that all I need to do is submit a PR with base information such as example sentences and stopwords (eg like [sl here](https://github.com/explosion/spaCy/tree/1caa2d1d16babb43b346e3eebcf229367bcc47f5/spacy/lang/sl))?

Thanks!

**Comment by polm** [comment]
Yes, if you don't need any tokenizer customization then a PR with data like in the Slovenian directory is fine.

**Reply by drvenabili** [reply]
Thanks. PR is here: https://github.com/explosion/spaCy/pull/11193
